index.js

```diff
@@ -123,10 +123,14 @@ function createIdObjects(row, idObject, importantEvents) {
 }
 
 function filterContributorByTime(idObject, dates) {
-  const timeWindow = moment.range([moment.utc(`${dates[0]}`, 'YYYY-MM-DD hh:mm'), moment.utc(`${dates[1]}`, 'YYYY-MM-DD hh:mm')]);
-  const contribsByUser = [];
+  const startDate = dates[0];
+  const endDate = dates[1];
+  const startMoment = moment.utc(`${startDate}`, 'YYYY-MM-DD hh:mm');
+  const endMoment = moment.utc(`${endDate}`, 'YYYY-MM-DD hh:mm');
+
+  const timeWindow = moment.range([startMoment, endMoment]);
   for(let i=0; i<idObject.contributions.length; i++) {
-    let contribDate = moment(idObject.contributions[i].created_at, "YYYY-MM-DD, h:mm:ss a")
+    let contribDate = moment.utc(idObject.contributions[i].created_at)
```

Comment:
For perfect consistency, this should probably be named contribMoment. I think we should just merge it as is. If you disagree, feel free to rename it. Either way, let me know when you want me to hit the merge button.

Reply:
changed to momentOfContribution after discussion outside of this comment chain